app/src/main/java/com/brainwallet/tools/manager/SyncManager.java

```diff
@@ -162,6 +135,16 @@ public void run() {
                         progressStatus = BRPeerManager.syncProgress(startHeight);
                         if (progressStatus == 1) {
                             running = false;
+                            /// Record sync time
```

Comment:
@andhikayuana this is my attempt to collect this sync info....is it correct?
The ideal thing is we compare the start and end times and show the duration in Settings 
We ALSO send the metadata as a custom event data into Firebase (issue: https://github.com/gruntsoftware/internal/issues/127).